src/bitmessageqt/messageview.py

```diff
@@ -49,11 +50,15 @@ def wheelEvent(self, event):
         """Mouse wheel scroll event handler"""
         # super will actually automatically take care of zooming
         super(MessageView, self).wheelEvent(event)
-        if (QtGui.QApplication.queryKeyboardModifiers() &
-                QtCore.Qt.ControlModifier) == QtCore.Qt.ControlModifier and event.orientation() == QtCore.Qt.Vertical:
+        if (
+            QtGui.QApplication.queryKeyboardModifiers() & QtCore.Qt.ControlModifier
+        ) == QtCore.Qt.ControlModifier and event.orientation() == QtCore.Qt.Vertical:
             zoom = self.currentFont().pointSize() * 100 / self.defaultFontPointSize
             QtGui.QApplication.activeWindow().statusBar().showMessage(
-                QtGui.QApplication.translate("MainWindow", "Zoom level %1%").arg(str(zoom)))
+                QtGui.QApplication.translate("MainWindow", "Zoom level %1%").arg(
```

Comment:
Don't care, up to @g1itch

Reply:
line too long complain was error to fix it I have made such changes

Reply:
Line too long is because there should be `_translate()` imported from `tr`, not `QtGui.QApplication.translate()`. I also don't care, because I rewritten this module in qt5-wip branch like most of `bitmessageqt`: https://github.com/g1itch/PyBitmessage/blob/qt5-wip/src/bitmessageqt/messageview.py

Reply:
what I should do for now undo it and leave as line too long?

Reply:
@coolguy-cell Change `QtGui.QApplication.translate` to `_translate` imported from `tr`.